test/e2e/shared/aws.go

```diff
@@ -141,11 +142,12 @@ func deleteResourcesInCloudFormation(prov client.ConfigProvider, t *cfn_bootstra
 		tayp := val.AWSCloudFormationType()
 		if tayp == configservice.ResourceTypeAwsIamRole {
 			role := val.(*cfn_iam.Role)
-			iamSvc.DeleteRole(&iam.DeleteRoleInput{RoleName: aws.String(role.RoleName)})
+			_, err := iamSvc.DeleteRole(&iam.DeleteRoleInput{RoleName: aws.String(role.RoleName)})
+			Expect(err).NotTo(HaveOccurred())
 		}
 		if val.AWSCloudFormationType() == "AWS::IAM::InstanceProfile" {
 			profile := val.(*cfn_iam.InstanceProfile)
-			iamSvc.DeleteInstanceProfile(&iam.DeleteInstanceProfileInput{InstanceProfileName: aws.String(profile.InstanceProfileName)})
+			_, _ = iamSvc.DeleteInstanceProfile(&iam.DeleteInstanceProfileInput{InstanceProfileName: aws.String(profile.InstanceProfileName)})
```

Comment:
Just out of curiosity, why returned error here is silenced?

Reply:
Failing on error here does not give any benefit, it will panic and not continue cleaning up the remaining resources down the function.
Instead of failing, we could add retry mechanism for all but best way is to have Boskos do the remaining clean up for us, which we have: https://github.com/kubernetes-sigs/boskos/blob/master/aws-janitor/resources/iam_instance_profiles.go

Reply:
thanks for explanation